asv_bench/benchmarks/pandas/frame_methods.py

```diff
@@ -0,0 +1,530 @@
+import string
```

Comment:
this is too big and some of functions already in testing - @jbrockmendel please make sure we don't have duplicates

Reply:
done